chain/chain/src/chain.rs

```diff
@@ -1,58 +1,60 @@
-use std::collections::{HashMap, HashSet};
-use std::sync::Arc;
-use std::time::{Duration as TimeDuration, Instant};
+use std::{
+    collections::{HashMap, HashSet},
```

Comment:
Should this also become
```
use std::{
    collections::{
        HashMap,
        HashSet,
    }
}
```

Reply:
That's only if I apply the second rule `imports_layout`.

Reply:
I beg not to do it! It is especially ugly for `std` imports. There was a discussion recently on reddit about it: https://www.reddit.com/r/rust/comments/kmw6xc/code_style_question_grouping_of_use_statements/

Reply:
My rational is: it is easier to read a single line than to follow multiline statements, and this way you hit merged conflicts less frequently.